docs/migrate-to-managed-asm/service-entry.yaml

```diff
@@ -0,0 +1,32 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START servicemesh_migrate_to_managed_asm_service_entry_serviceentry_external_service_http]
+# With this ServiceEntry, we're instructing the mesh to
+# direct HTTP requests (from the mesh) for v2.example.com to the specified IP address.
```

Comment:
nit: rewording this comment
`This ServiceEntry instructs the mesh to direct HTTP requests external from the mesh for v2.example.com to the specified IP address.`

Reply:
Thanks, Xtine. Your wording flows better.
I've updated the comment. 
(I've also updated my confusing wording — `for v2.example.com` is now `that target v2.example.com`.)